scripts/pi-hole/php/database.php

```diff
@@ -81,6 +85,10 @@ function add_to_table($db, $table, $domains, $wildcardstyle=false)
 	$num = 0;
 	foreach($domains as $domain)
 	{
+		// Skip empty domains
+		if(strlen($domain) == 0)
+			continue;
```

Comment:
This should be an impossible case now that the input domain list is properly split. It can be assumed that the input domain list contains valid data.